pkg/watcher/template_watch_poll.go

```diff
@@ -1,12 +1,21 @@
 package watcher
 
 import (
+	"crypto/md5"
+	"encoding/hex"
 	"github.com/golang/glog"
 	"io/ioutil"
 	"os"
 	"time"
 )
 
+const (
+	// how often to check for template file changes
+	POLL_INTERVAL_SECONDS = 15
+	// how often print template info for troubleshooting
+	TIMESTAMP_DISPLAY_INTERVAL_SECONDS = 3600
```

Comment:
We can use the `time.Duration` type here, directly.

```suggestion
	// how often to check for template file changes
	POLL_INTERVAL = 15 * time.Second
	// how often print template info for troubleshooting
	TIMESTAMP_DISPLAY_INTERVAL = 1 * time.Hour
```

Reply:
I applied both requested changes, pls review again